nvidia/nvidia-utils/nvidia-utils.install

```diff
@@ -1,13 +1,26 @@
+post_install() {
+  # Enable NVIDIA Services at first installation
+  # The services are mandatory, see under systemd configuration
+  # https://download.nvidia.com/XFree86/Linux-x86_64/560.35.03/README/powermanagement.html#SystemdConfigur74e29
+  # This is also an requirement to have sleep working together with PreserveAllocations
+  # https://gitlab.archlinux.org/archlinux/packaging/packages/nvidia-utils/-/commit/55644f78820fd382fbdf283b1fd7f08e6b7c22d7
+  # https://gitlab.archlinux.org/archlinux/packaging/packages/nvidia-utils/-/merge_requests/16
+  systemctl enable nvidia-resume nvidia-hibernate nvidia-suspend
+}
 
 post_upgrade() {
-  echo "If you run into trouble with CUDA not being available, run nvidia-modprobe first."
-  echo "If you use GDM on Wayland, you might have to run systemctl enable --now nvidia-resume.service"
+  # Only enable the services for the 560.35.03-16 version
+  # This avoids, that the services are automatically enabled in every upgrade
+  if (( $(vercmp $2 560.35.03-16) < 0)); then
+    for service in nvidia-resume nvidia-hibernate nvidia-suspend; do
+        if ! systemctl is-enabled --quiet $service; then
+            echo "Enabling $service..."
+            systemctl enable $service
+        fi
+    done
+  fi
 }
-# For the latter nore, see also https://gitlab.gnome.org/GNOME/gdm/-/issues/784
-
-post_install() {
-  post_upgrade
 
-    echo "Enabling services..."
-    systemctl enable nvidia-resume nvidia-hibernate nvidia-suspend
+post_remove() {
+  systemctl disable nvidia-resume nvidia-hibernate nvidia-suspend
```

Comment:
What's the point of disabling services when removing a package if they are contained within nvidia-utils?